js/screens/ActivityComplete/styles.js

```diff
@@ -0,0 +1,72 @@
+import { StyleSheet, Dimensions } from "react-native";
+const { height, width } = Dimensions.get("window");
+const styles = StyleSheet.create({
+  background: {
+    width: width,
+    height: height
```

Comment:
if the key and value both are same you can just type height and width and it will work.